lang/cs/Org.Apache.REEF.Wake/Remote/Impl/WritableTransportClient.cs

```diff
@@ -44,12 +46,14 @@ public class WritableTransportClient<T> : IDisposable where T : IWritable
         /// Used to send messages to the specified remote endpoint.
         /// </summary>
         /// <param name="remoteEndpoint">The endpoint of the remote server to connect to</param>
-        public WritableTransportClient(IPEndPoint remoteEndpoint)
+        /// <param name="injector">The injector to pass arguments to incoming messages</param>
+        public WritableTransportClient(IPEndPoint remoteEndpoint, IInjector injector)
         {
             Exceptions.ThrowIfArgumentNull(remoteEndpoint, "remoteEndpoint", Logger);
 
-            _link = new WritableLink<T>(remoteEndpoint);
+            _link = new WritableLink<T>(remoteEndpoint, injector);
             _cancellationSource = new CancellationTokenSource();
+            _injector = injector;
```

Comment:
No, I have removed it.

On Wed, May 13, 2015 at 11:59 AM, Julia Wang <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.Wake/Remote/Impl/WritableTransportClient.cs
> <https://github.com/apache/incubator-reef/pull/181#discussion_r30263987>:
>
> >              _cancellationSource = new CancellationTokenSource();
> > +            _injector = injector;
>
> Is the _injector used anywhere else that you need to keep an instance
> variable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/181/files#r30263987>.
>